static/js/start.js

```diff
@@ -10,16 +10,16 @@ async function startUp(){
 
 function addListeners(){
     window.onmessage = function (e) {
-        if (e.data == 'showSpinner') {
+        if (e.data === 'showSpinner') {
```

Comment:
Is data type a necessary consideration in these tests?  If not, == is fine... and I tend to think data type is not a necessary consideration.  

Also, is this a page generated by the group, or does this come from the library?